packages/react/package.json

```diff
@@ -20,7 +20,7 @@
     "@heliofi/sdk": "^1.0.2",
     "@heliofi/solana-adapter": "^4.0.2",
     "@popperjs/core": "^2.11.7",
-    "@project-serum/anchor": "^0.25.0",
+    "@project-serum/anchor": "^0.26.0",
```

Comment:
do we need the project serum ?

Reply:
We don't I updated just to have the same version as in BE if we ever need